elrond_dex_farm/src/lib.rs

```diff
@@ -536,14 +543,16 @@ pub trait Farm {
         }
     }
 
-    #[view(simulateEnterFarm)]
+    #[endpoint(simulateEnterFarm)]
```

Comment:
simulateEnterFarm should only calculate the minted rewards and not actually mint them